### PR TITLE
Add some text under the form header

### LINKED ISSUE #182: [USABILITY]: Application saved message could be missed by assistive devices
## Issue
**UPDATE 3/30/2020** Revising assignees to raise awareness with DEPO leads and VSP design leadership. This is an exploratory issue, but one that could have impact for all forms that can be saved and continued later.

The `Application has been saved` message is appended after the `Back` and `Continue` buttons. This means the message may be missed by assistive device users, who would have a high chance of moving to the next page before having it read aloud. This ticket is not a defect, but only an item to investigate about placing the message before the buttons, or considering appending it to the beginning of next screens, as a way to have it read on first load.

/cc @emilywaggoner for design guidance. This item was flagged as an improvement/enhancement by the 508 office. They're open to seeing what new guidance we come up with and happy to help us re-test.

## Acceptance Criteria

As a screenreader user, I would like to hear the form saved announcement periodically. This could be via `aria-live` attributes, or by positioning it before the buttons and adding a `tabindex="0"` attribute.

## Environment

* Windows 7 64 Bit Enterprise Edition SP1
* Google Chrome
* JAWS 2018.0

---

screen shot 2018-08-22 at 10 58 01 pm

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/182

## Testing done
:eyes: 

## Screenshots
image

### CODE REVIEW

**Comment by rtwell** [comment]
I can't quite tell from the screen grab—I wanna make sure the typography is using our gray-dark color (like the rest of our type)

**Comment by cvalarida** [comment]
It is not currently. I put the `vads-u-color--gray-medium` class on it. I'll take it off. :+1:

**Comment by cvalarida** [comment]
Updated the font color.
image

**Review by bkjohnson — APPROVED** [review]

**Review by rtwell — APPROVED** [review]
lgtm!

**Review by 1Copenut — APPROVED** [review]
LGTM! I spot checked the branch on `localhost` with MacOS Safari + VoiceOver. The heading and span text was read as expected. We should do a brief spot check in Staging, but won't need more than 5-10 minutes.

**Comment by Mottie** [comment]
Updated to add the application ID to the header

Screen Shot 2021-08-10 at 2 00 49 PM

It matches the ID in the green box below the back & continue button

**Comment by Mottie** [comment]
Oops, the "Your application will be saved..." will show when you're not logged in & filling out a form that doesn't need authentication, e.g. https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction

Will update to check the logged in state.

**Comment by cvalarida** [comment]
Thanks for the addition, @Mottie! :raised_hands: I just added a period to the end of the sentence, but this looks great. Checking for `isLoggedIn` is perfect. I didn't consider not showing that message to users who aren't logged in, but what you've got there is correct. Because we won't save the data for unauthenticated users. :+1:

File: src/applications/pensions/tests/pensions.cypress.spec.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const fillAdditionalSources = ({ additionalSources }) => {
   cy.findByText(/add another/i).click();
   cy.findByLabelText(/source/i).type(name);
   cy.findByLabelText(/amount/i).type(amount);
-  cy.findByText(/save/i).click();
+  cy.findByText(/^Save$/).click();
```
Comment (cvalarida) [inline src/applications/pensions/tests/pensions.cypress.spec.js:17]: The heading now contains the text `Your application will be saved on every change.`, so this selector matched both the heading _and_ the button it should match. Making this selector more specific finds only the button again.

**Review by theodur — APPROVED** [review]

**Comment by cvalarida** [comment]
Thanks, @theodur! :raised_hands:

**Pull request merged**